snowddl/converter/task.py

```diff
@@ -0,0 +1,92 @@
+from re import compile, DOTALL
+
+from snowddl.blueprint import ObjectType
+from snowddl.converter.abc_converter import ConvertResult
+from snowddl.converter.abc_schema_object_converter import AbstractSchemaObjectConverter
+from snowddl.converter._yaml import YamlLiteralStr
+from snowddl.parser.task import task_json_schema
+import json
+
+task_text_re = compile(r"^.*\n\)\sas(.*)$", DOTALL)
+after_syntax_re = compile(r"^(\w+)?\((.*)\)$")
```

Comment:
These regular expressions do not seem to be used.

Reply:
Done.